spec/services/update_duplicate_matches_spec.rb

```diff
@@ -39,6 +39,22 @@
           expect(candidate2.reload.submission_blocked).to be(false)
         end
       end
+
+      context 'when the candidates have been mannually resolved and new candidate is added to the match' do
+        let(:candidate3) { create(:candidate, email_address: 'exemplar3@example.com') }
+
+        before do
+          FraudMatch.update_all(resolved: true)
```

Comment:
Is there a reason for updating all, or can we just update the first match?

Reply:
I did for convenience but am happy to do only for the first one.